src/app/collective-rewards/utils/getMostAdvancedProposal.ts

```diff
@@ -4,7 +4,8 @@ import { ProposalState } from '@/shared/types'
 
 const inactiveProposalsStates = [ProposalState.Canceled, ProposalState.Defeated, ProposalState.Expired]
 const isActive = (state: ProposalState) => !inactiveProposalsStates.includes(state)
-const isBuilderActive = (status?: BuilderStateFlags) => !status || status.activated
+const isBuilderActive = (status?: BuilderStateFlags) =>
+  !status || status.activated || (!status.activated && status.communityApproved)
```

Comment:
why not just `!status || status.activated || status.communityApproved`?
Since if `status.activated` fails, the `!status.activated` will be already true.

Reply:
Right, thank you 🙏